### DISCUSSION TITLE
websocket support

### DISCUSSION THREAD

**Discussion by scriptyyy13** [deta/cloud-docs#495] [Ideas]
Thanks to the addition of websocket support, it will be possible to implement a bunch of cool features!

so, pls, add websocket support to micros.

**Comment by AyushSehrawat** [comment]
But there won't be any websocket support

**Reply by Sokhavuth** [reply]
Great news to hear that Deta supports websocket. Could you put a link to this good news please.

**Reply by scriptyyy13** [reply]
this is a suggestion to add websocket support
